### DISCUSSION TITLE
```net.8.0``` ```RuntimeIdentifier``` errors on Windows

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#13648] [General]
I've created the UnoApplication with **.net8.0**

![image](https://github.com/unoplatform/uno/assets/69314237/ca2ca75a-c67f-4e4f-bec9-5e85546251ad)

![image](https://github.com/unoplatform/uno/assets/69314237/47a92f74-1d7e-4a29-82e9-da614534c520)

<h3>What I already tried</h3>

```XML
  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0-windows10.0.19041'">
    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
  </PropertyGroup>
```

This lines didn't helps me.

Also I updated Uno.WinUI packages to ```5.0.0-dev.2593```.
It didn't helps me also.

[UnoRuntimeIdentifierErrors.zip](https://github.com/unoplatform/uno/files/12617557/UnoRuntimeIdentifierErrors.zip)

It looks similar as the following issue:
https://github.com/unoplatform/uno.templates/issues/183

**Comment by jeromelaban** [comment]
Thanks for the report. Which version of .NET 8 are you using?

**Reply by limeniye** [reply]
latest preview version
[8.0.100-rc.1.23455.8](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)

**Comment by VincentH-Net** [comment]
I ran into the same problem today (also using same version as @limeniye) - does Uno work with .NET 8 RC1?

I tried [this workaround](https://github.com/dotnet/maui/issues/17330#issuecomment-1716593275) which was advised for the same error on Maui but to no avail

**Comment by VincentH-Net** [comment]
Found a workaround in [WindowsAppSDK issue 3842](https://github.com/microsoft/WindowsAppSDK/issues/3842#issuecomment-1723293055): add below to the Windows csproj
```xml
	<PropertyGroup>
		<UseRidGraph>true</UseRidGraph>
	</PropertyGroup>
```

**Reply by VincentH-Net** [reply]
A warning: this workaround has a cascading effect; you may need to apply it to any referenced projects as well.

When you apply it to all relevant projects (for the Windows build), the Windows project builds but I gave up before I got through solving all build & runtime issues in the template wizard-generated solution (with updates to latest preview Uno packages). I opted to recreate in .NET 7 and will wait for next Uno / .NET 8 release to retry

**Comment by limeniye** [comment]
Fixed https://github.com/unoplatform/uno/pull/13700#event-10421700559
